profile.html

```diff
@@ -0,0 +1,220 @@
+          <div class="card--profile">
+              <img class="img-profile" src="https://source.unsplash.com/500x500/?faces" alt="">
+            <h1 class="card--name">Profile Name</h1>
+            <h2 class="card--city">Stadt, Land</h2>
+            <p class="card--slogan">You have to taste to know it's waste.</p>
+        </div>
+        <div class="card-social-infos">
+            <ul class="card-list-item">
+                <li>2,467</li>
+                <li class="card-list-item--li">Followers</li>
+            </ul>
+            <ul class="card-list-item">
+                <li>1,589</li>
+                <li class="card-list-item--li">Following</li>
+            </ul>
+            <p class="card-edit-profile-button">Edit Profile</p>
+        </div>
+        <ul class="filter-posts">
+            <li class="text--underline">Posts</li>
+            <li class="text--dark-grey">Stories</li>
+            <li class="text--dark-grey">Liked</li>
+            <li class="text--dark-grey">Tagged</li>
+        </ul>
+      </section>
+      <section class="post">
+        <div class="avatar">
+          <img
+            class="avatar__image"
+            src="https://source.unsplash.com/100x100/?faces"
+            alt=""
+          />
+          <h2 class="avatar__heading">Jacob Washington</h2>
+          <span class="avatar__timespan">20m ago</span>
+        </div>
+        <span class="post__burger-menu"></span>
+        <div class="post__text">
+          <q
+            >If you think you are too small to make a difference, try sleeping
+            with a mosquito.</q
+          >
+          <p>~ Dalai Lama</p>
+        </div>
+        <div class="post__icons">
+          <span class="post__icon--thumb mr-1rem">2,245</span>
+          <span class="post__icon--bubble mr-1rem">45</span>
+          <span class="post__icon--share mr-1rem">124</span>
+        </div>
+        <svg
+          aria-hidden="true"
+          focusable="false"
+          data-prefix="far"
+          data-icon="bookmark"
+          class="post__bookmark svg-inline--fa fa-bookmark"
+          role="img"
+          xmlns="http://www.w3.org/2000/svg"
+          viewBox="0 0 384 512"
+        >
+          <path
+            fill="currentColor"
+            d="M336 0h-288C21.49 0 0 21.49 0 48v431.9c0 24.7 26.79 40.08 48.12 27.64L192 423.6l143.9 83.93C357.2 519.1 384 504.6 384 479.9V48C384 21.49 362.5 0 336 0zM336 452L192 368l-144 84V54C48 50.63 50.63 48 53.1 48h276C333.4 48 336 50.63 336 54V452z"
+          ></path>
+        </svg>
+      </section>
+
+      <section class="post">
+        <div class="avatar">
+          <img
+            class="avatar__image"
+            src="https://source.unsplash.com/100x100/?faces"
+            alt=""
+          />
+          <h2 class="avatar__heading">Tony Monta</h2>
+          <span class="avatar__timespan">1h ago</span>
+        </div>
+        <span class="post__burger-menu"></span>
+        <div class="post__text">
```

Comment:
Do you really need this div around your p?

Reply:
Considering the fact that p is a block element, we do not need the div around it. But we forgot about it when writing the html.